.golangci.yml

```diff
@@ -16,9 +16,7 @@ linters:
     - exportloopref
     - forcetypeassert
     - goconst
-    - gofmt
-    - goimports
-    # - gofumpt
+    - gofumpt
```

Comment:
I would like to suggest not enabling gofumpt. AFAIK, we have team members who don't like its strictness and disable it locally.

As a result, if we use different linters, reviewing each other PRs becomes more painful since you end up auto-linting someone else's PR by accident while reviewing.

For this reason I would like to suggest, not enabling it

Reply:
hmmmm well in that case, I'll close the PR I suppose.